en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md

```diff
@@ -637,29 +637,98 @@ public function countDuplicates($model, $fieldToCheck)
 }
 ```
 
+### Common Table Expressions (CTEs aka the `WITH` clause) {#cte}
```

Comment:
I'm intentionally cramming the common three ways to refer to this functionality into the heading for search discovery purposes:
- "Common Table Expressions"
- "CTE"
- "WITH clause"